301-sf-preview-windows-containers/azuredeploy.json

```diff
@@ -0,0 +1,565 @@
+          ],
+          "defaultValue": "None",
+          "metadata": {
+              "description": "Protection level.Three values are allowed - EncryptAndSign, Sign, None. It is best to keep the default of EncryptAndSign, unless you have a need not to"
+          }
+      },
+      "clusterReliabilityLevel": {
+          "type": "string",
+          "defaultValue": "Silver",
+          "allowedValues": [
+              "Silver",
+              "Platinum"
+          ],
+          "metadata": {
+              "description": "Reliability level for the Service Fabric cluster."
+          }
+      },
+      "nt0InstanceCount": {
+          "type": "int",
+          "defaultValue": 5,
+          "metadata": {
+              "description": "Instance count for node type"
+          }
+      }
+  },
+  "variables": {
+      "clusterLocation": "[resourceGroup().location]",
+      "computeLocation": "[resourceGroup().location]",
+      "dnsName": "[parameters('clusterName')]",
+      "vmName": "vm",
+      "publicIPAddressName": "PublicIP-VM",
+      "publicIPAddressType": "Dynamic",
+      "vmStorageAccountContainerName": "vhds",
+      "virtualNetworkName": "VNet",
+      "addressPrefix": "10.0.0.0/16",
+      "nicName": "NIC",
+      "lbName": "LoadBalancer",
+      "lbIPName": "PublicIP-LB-FE",
+      "availSetName": "AvailabilitySet",
+      "maxPercentUpgradeDomainDeltaUnhealthyNodes": "100",
+      "vnetID": "[resourceId('Microsoft.Network/virtualNetworks', variables('virtualNetworkName'))]",
+      "overProvision": "false",
+      "nt0applicationStartPort": "20000",
+      "nt0applicationEndPort": "30000",
+      "nt0ephemeralStartPort": "49152",
+      "nt0ephemeralEndPort": "65534",
+      "nt0fabricTcpGatewayPort": "19000",
+      "nt0fabricHttpGatewayPort": "19080",
+      "subnet0Name": "Subnet-0",
+      "subnet0Prefix": "10.0.0.0/24",
+      "subnet0Ref": "[concat(variables('vnetID'),'/subnets/',variables('subnet0Name'))]",
+      "lbID0": "[resourceId('Microsoft.Network/loadBalancers',concat('LB','-', parameters('clusterName'),'-',variables('vmNodeType0Name')))]",
+      "lbIPConfig0": "[concat(variables('lbID0'),'/frontendIPConfigurations/LoadBalancerIPConfig')]",
+      "lbPoolID0": "[concat(variables('lbID0'),'/backendAddressPools/LoadBalancerBEAddressPool')]",
+      "lbProbeID0": "[concat(variables('lbID0'),'/probes/FabricGatewayProbe')]",
+      "lbHttpProbeID0": "[concat(variables('lbID0'),'/probes/FabricHttpGatewayProbe')]",
+      "lbNatPoolID0": "[concat(variables('lbID0'),'/inboundNatPools/LoadBalancerBEAddressNatPool')]",
+      "vmNodeType0Name": "[toLower(concat('NT1', variables('vmName')))]",
+      "vmNodeType0Size": "Standard_DS2_v2",
+      "autoScaleSettingsName": "[concat(variables('vmNodeType0Name'), 'AutoScaleSettings')]",
+      "startScriptLocation": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/301-sf-preview-windows-containers/scripts/setup-docker.ps1"
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc) for how to handle artifacts